isd-app/src/App.jsx

```diff
@@ -4,6 +4,9 @@ import LogIn from "./pages/LogIn/Login";
 import SignUp from "./pages/SignUp/SignUp";
 import Request from "./pages/Request/Request";
 import CourseRequest from "./pages/CourseRequest/CourseRequest";
+import AccounSetUpEmail from "./pages/AccountSetUpEmail/AccountSetUpEmail";
+import AccountSetUpNamePassword from "./pages/AccountSetUpNamePassword/AccountSetUpNamePassword";
```

Comment:
I am not sure about the names. These three pages are a part of one process. Right now I came up with the idea of putting all of them under the one directory ./pages/AccountSetUp:

./pages/AccountSetUp / Email / Email.jsx
./pages/AccountSetUp / NamePassword / NamePassword.jsx
./pages/AccountSetUp / CompanyName / CompanyName.jsx

Will it be better?

Reply:
Yes, that'd be better.